### ISSUE TITLE
Background color isn't applied to whole Donation Form

### ISSUE THREAD

**Issue by Himani3010** [Charitable/Reach#61]
![image](https://user-images.githubusercontent.com/22770041/49926063-be155a80-fee0-11e8-966e-4c67991e49bf.png)

Reference link: http://demo.wpcharitable.com/reach/campaigns/unicef-syrian-crisis-appeal/donate/

**Comment by ericnicolaas** [comment]
@Himani3010 This is a bug, though what we actually want here is for that light grey background to extend over just the "Your Details" section of the form. i.e. We don't want the background colour applied to the whole donation form, just that section. 

This seems to fix the issue:

```
#charitable-user-fields,
#charitable-gateway-fields .charitable-gateway-fields {
    float: left;
}
```

**Comment by ericnicolaas** [comment]
Closed by 24675d8
